content/en/docs/tasks/traffic-management/locality-load-balancing/before-you-begin/index.md

```diff
@@ -6,6 +6,35 @@ keywords: [locality,load balancing,priority,prioritized,kubernetes,multicluster]
 test: yes
 owner: istio/wg-networking-maintainers
 ---
+
+{{< tip >}}
+In a single multi-zone cluster environment, locality load balancing can also be configured for failover to a different zone within the same cluster.
+To test it, you will need to create a cluster with multiple worker zones and deploy an istiod instance and the app to each zone.
+
+1) If you don’t have a multi-zone Kubernetes cluster, you can deploy one locally using `kind` with the following command:
+
+{{< text syntax=bash snip_id=none >}}
+$ kind create cluster --config=- <<EOF
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+  - role: worker
+  - role: worker
+  - role: worker
+EOF
+{{< /text >}}
+
+2) Use `topology.kubernetes.io/zone` to label each worker with a zone name:
+
+{{< text syntax=bash snip_id=none >}}
+$ kubectl label node kind-worker topology.kubernetes.io/zone=us-south10
+$ kubectl label node kind-worker2 topology.kubernetes.io/zone=us-south12
+$ kubectl label node kind-worker3 topology.kubernetes.io/zone=us-south13
+{{< /text >}}
```

Comment:
nit: a little better format for the commands:
```suggestion
    {{< text syntax=bash snip_id=none >}}
    $ kind create cluster --config=- <<EOF
    kind: Cluster
    apiVersion: kind.x-k8s.io/v1alpha4
    nodes:
    - role: control-plane
    - role: worker
    - role: worker
    - role: worker
    EOF
    {{< /text >}}

2) Use `topology.kubernetes.io/zone` to label each worker with a zone name:

    {{< text syntax=bash snip_id=none >}}
    $ kubectl label node kind-worker topology.kubernetes.io/zone=us-south10
    $ kubectl label node kind-worker2 topology.kubernetes.io/zone=us-south12
    $ kubectl label node kind-worker3 topology.kubernetes.io/zone=us-south13
    {{< /text >}}
```

Reply:
When I commit this change and run the page locally, I see some commands are not in the text block. I have tried with different indents but could not get the correct format. Any suggestions to this? 
![Screenshot 2023-09-05 at 11 09 25 AM](https://github.com/istio/istio.io/assets/36289669/85e0a8db-7964-4050-a392-f4802cacc95b)